src/FwdState.h

```diff
@@ -195,7 +196,8 @@ class FwdState: public RefCountable, public PeerSelectionInitiator
 
     HappyConnOpenerPointer connOpener; ///< current connection opening job
     ResolvedPeersPointer destinations; ///< paths for forwarding the request
-    Comm::ConnectionPointer serverConn; ///< a successfully opened connection to a server.
+    Comm::ConnectionPointer serverConn; ///< a successfully opened connection to a server
```

Comment:
Let's undo this correct but out-of-scope change to avoid opening the Pandora box of perfecting documentation of this data member. Sorry if I created this change.

```suggestion
    Comm::ConnectionPointer serverConn; ///< a successfully opened connection to a server.
```